homeworkArraysFunctions.ts

```diff
@@ -0,0 +1,114 @@
+//1.	 Create an array of strings and add a new element at the end of the array. Log the result.
+
+let bookGenre: string[] = ["Criminal", "Detective","Romance","Historical"];
+bookGenre.push("Horror");
+
+console.log("bookGenre", bookGenre);
+
+
+//2.	Create an array of numbers and remove the first element from the array. Log the result.
+
+let numberArray: number[] = [15,20,25,30,35];
+numberArray.pop();
+console.log("numberArray", numberArray);
+
+//3.	 Use the map method to create a new array and divide each number by 2  “num / 2”   from [1, 2, 3, 4, 5].  Log the result.
+
+let baseNumArr: number []= [1,2,3,4,5];
+function numdivision(element:number) :number {
+    return element/2;    
+}
+let mapedNumArrFunc = baseNumArr.map(numdivision);
+console.log("mapedNumArrFunc",mapedNumArrFunc);
+
+let mappedNumArr = baseNumArr.map((element:number) => element/2);
+console.log("mappedNumArr", mappedNumArr)
+
+//4.	 Use the filter method to create a new array containing only numbers greater than 5 from [3, 7, 1, 9, 12, 4]. Log the result.
+
+let noFilteredNumArr: number [] = [3, 7, 1, 9, 12, 4];
+let filteredNumArr= noFilteredNumArr.filter((element: number) => (element > 5 ? element : 0));
```

Comment:
You don't need the ternary operator check:
noFilteredNumArr.filter((element: number) => element > 5 );